app/site/app/assets/javascripts/site/components/data-table.js

```diff
@@ -1,13 +1,30 @@
 $(function() {
+  function dataRequest(url, binding) {
+    $.ajax({
+      url: url,
+      dataType: 'jsonp',
+      success: function (data) {
+        this.setState({data: data});
+      }.bind(binding),
+      error: function (xhr, status, err) {
+        console.error(url, status, err.toString());
+      }.bind(binding)
+    });
+
+  }
 
   var DataTableRow = React.createClass({
     render: function () {
       var self = this;
       var fields = this.props.fields.map(function (field) {
-        var val = self.props.row[field.name],
+        var val,
           key = self.props.row.id + '-' + field.name;
 
-        if(field.method) { val = field.method(self.props.row) }
+        if(field.method) {
```

Comment:
We may want to avoid this if we want to use the PureMixin (speed reasons), If statements are bad!